README.rst

```diff
@@ -134,3 +137,10 @@ Contrast that with the Python standard library's ``urlib.parse`` module:
     'www.googlé.com'
     >>> parsed_url.path
     '/./path/../path2/'
+
+Alternative (can_ada)
+-----------
+
+The ada-python bindings are built on CFFI, an approach that has the binding between C and Python written in Python itself, which loses some of the performance benefits of using Ada in the first place when most of the time is spent just making the function call.
+
+The can_ada (Canadian Ada) is a different Python binding that uses pybind11 to generate the binding code, which is then compiled into a Python extension module. This approach has the potential to be much faster than the ada-python bindings. See  https://tkte.ch/articles/2024/03/15/parsing-urls-in-python.html
```

Comment:
Perhaps we could use Sphinx markdown for the link here?

Reply:
Can you be specific?

Reply:
There is a bare URL here that we could make a hyperlink by using Sphinx markup - there are examples above in the document.

I can change it in a separate PR.

Reply:
great